src/main/features/response/views/check-and-send.njk

```diff
@@ -123,6 +123,16 @@
 
     {% endif %}
 
+    {% if draft.isResponsePartiallyRejectedDueTo(RejectPartOfClaimOption.AMOUNT_TOO_HIGH) %}
+      {{ tableStart('whenWillYouPay', 'When will you pay?') }}
+
+      {{ row('When will you pay', t(draft.defendantPaymentOption.option.value) | capitalize, paths.defenceFullPartialPaymentOptionsPage.evaluateUri({ externalId: claim.externalId })) }}
```

Comment:
be careful of using the capitalise filter it doesn't always do what you want, better to use a `displayValue`